.github/workflows/cifuzz.yml

```diff
@@ -95,7 +101,11 @@ jobs:
             while [[ $(date -u +%s) -le $endtime ]]
             do
                 echo "Time Now: `date +%H:%M:%S`"
-                python3 scripts/run_fuzzer.py --no_checks --${{ matrix.fuzzer }} --${{ matrix.data }} --max_queries=1000 --shell=../duckdb
+                if [ "${{ matrix.enable_verification }}" == "true" ]; then
+                  python3 scripts/run_fuzzer.py --no_checks --${{ matrix.fuzzer }} --${{ matrix.data }} --enable_verification --max_queries=1000 --shell=../duckdb
```

Comment:
I should have mentioned this before I merged the other PR, but it might be better if `enable_verification`  is passed like this

`--enable_verification=true` or `--enable_verification=false`, that way you don't need to `if [] ... else ... fi`

Reply:
Did it - now it passes the `--enable_verification` with its value, which got from the matrix